CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,4 @@ Add: monit_log_processing.py script to contextBroker RPM (Issue #1083)
 Add: httpsPrepare.sh script to contextBroker-tests RPM (Issue #2)
 Fix:  Fixed a bug about generic error handling for API version 2 (Issue #1087)
 Fix:  Fixed a bug about error handling of invalid Service-Path for API version 2 (Issue #1092)
+Add:  FIWARE::StringQuery scope, implementing filtering capabilities (equal, unqual, greater/lesser than, ranges, existence) (Issue #864)
```

Comment:
I would change *greater/lesser than* for *greater/less than*.

Why?
Well, we use "less than" for expressing "<".
Lesser than is correct English but it isn't used for expressing this concept.